packages/connect-cloudflare/conformance/client-worker.ts

```diff
@@ -49,7 +49,7 @@ export default createWorkerHandler({
           testName: req.testName,
         });
         try {
-          const invokeResult = await invoke(createTransport(req), req);
+          const invokeResult = await promiseInvoke(createTransport(req), req);
```

Comment:
Not sure if it is worth it to also test the callback client here in the client worker. It will be a bit more difficult to do this plus both types of clients are already being tested via the changes in `client.ts`.

Reply:
I think this is redundant, see https://github.com/connectrpc/connect-es/pull/1083#pullrequestreview-2093263022